### PR TITLE
update xenial64 DISTRO_PET_REPOS

### PR DESCRIPTION
**Pull request by peabee**

### CODE REVIEW

**Comment by wdlkmpx** [comment]
i can fix it for you if you want

**Comment by peabee** [comment]
That would be great - thanks

**Comment by wdlkmpx** [comment]
For bionic you can use artful's DISTRO_PET_REPOS. I also made some other changes, a few pkgs might be missing. just remove the entry from the pkgs specs.

Or you might want to delete artful now and add bionic so it can be edited programmatically. and you can open a pull request in case something is wrong or missing..

wary5 has too many unsuitable packages, 'common' was fixed a bit.

to get rid of wary5, i see these pkgs are missing from tahrpup:

WARNING: chmlib pkg was not found!
WARNING: diffutils pkg was not found!
WARNING: goffice pkg was not found!
WARNING: util-linux pkg was not found!
WARNING: wvstreams pkg was not found!

chmlib: can be removed
diffutils: must use compat distro version 
goffice: copy goffice-0.8.16-w5c.pet from wary5 to tahr
util-linux:  must use compat distro version 
wvstreams: copy wvstreams-4.6.1-w5.pet from wary5 to tahr

so basically copying goffice-0.8.16-w5c.pet and wvstreams-4.6.1-w5.pet from wary5 to tahr will make all the other changes possible..

i remember reading long ago that dvdauthor was broken in precise. newer versions are available in the ubuntu repos..

**Comment by wdlkmpx** [comment]
shouldn't it be ubuntu/bionic ?

**Comment by wdlkmpx** [comment]
i'll add your stuff but it currently looks "out of place", the
'eudev-1.6.3-i486' package shouldn't be  in Packages-extra-upupbb
(current eudev version 3.2.5), etc ..

so i'll remove some entries here and there.

**Comment by wdlkmpx** [comment]
also desk_icon_theme* are noarch packages, but i noticed that various
themes are only in certain pet repos, don't know why...

**Comment by peabee** [comment]
> shouldn't it be ubuntu/bionic

No!! Bionic pup should be a successor to xenial..... upupbb is different.

**Comment by wdlkmpx** [comment]
i meant Packages-puppy-upupbb-official .

ok, but i doubt bionicpup will be ever released.

**Comment by wdlkmpx** [comment]
Packages-puppy-precise-official can also be deleted if the following
pet pkgs are copied from precise to tahr:

dvdauthor-0.7-i486.pet
gnome-menus-2.14.3-up.pet
gnome-menus_DEV-2.14.3-up
gnumeric-1.10.17-i486-precise
gtkhash-0.6.0-i686-up.pet
osmo-0.2.10-patched_96dpi_version_libnotify-x86-precise.pet

**Comment by peabee** [comment]
All already there.....must be a package db problem if they're not getting picked up....
